core-it-support/maven-it-plugin-bootstrap/src/main/java/org/apache/maven/its/bootstrap/DownloadMojo.java

```diff
@@ -82,8 +86,10 @@
     @Override
     public void execute() throws MojoFailureException
     {
+        // this or that: either resolver file listed artifacts or collect lifecycle packaging plugins
         if ( file != null && file.exists() )
         {
+            System.out.println( "Collecting artifacts from file: " + file );
```

Comment:
Why system out instead of Mojo log?

Reply:
I just followed existing code, see below, in this mojo nothing uses logger, but everything uses System.out